modules/imgproc/test/ocl/test_imgproc.cpp

```diff
@@ -234,7 +234,7 @@ OCL_TEST_P(CornerMinEigenVal, Mat)
         OCL_OFF(cv::cornerMinEigenVal(src_roi, dst_roi, blockSize, apertureSize, borderType));
         OCL_ON(cv::cornerMinEigenVal(usrc_roi, udst_roi, blockSize, apertureSize, borderType));
 
-        Near(1e-5, true);
+        Near(0.1, true);
```

Comment:
Is it possible to add here some check to detect your platform / OpenCL device?
I would prefer to keep original check threshold for x86 platform.

Reply:
I was thinking to base the tests on the OpenCL spec rather than a specific implementation

Reply:
Not sure about what do you mean.

Does this work for you?

```
if (ocl::Device::getDefault().isIntel())
    Near(1e-5, true);
else
    Near(0.1, true);  // using native_* OpenCV functions may loose accuracy
```

Reply:
Yeah that works, I'll change it to be like that